Cakefile

```diff
@@ -129,6 +129,11 @@ task 'build:browser', 'rebuild the merged script for inclusion in the browser',
 
 task 'doc:site', 'watch and continually rebuild the documentation for the website', ->
   # Helpers
+  css = fs.readFileSync('./documentation/css/docs.css', 'utf-8') + '\n' +
+        fs.readFileSync('./documentation/css/tomorrow.css', 'utf-8')
```

Comment:
I like all the inlining stuff. But I really think we should minify the CSS first.